git-hg/update.sh

```diff
@@ -1,14 +1,29 @@
 #!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 
 echo "Common defs"
 
-. import-common.sh
+# shellcheck disable=SC1091
+source import-common.sh
 
 echo "Enter hg"
 
 cd hg || exit 1
 
-bpaths=`ls -d -1 */*`
+# shellcheck disable=SC2035
+bpaths=$(ls -d -1 */*)
```

Comment:
I also wasn't 100% sure that `ls -d -1 */*` lists just dirs (which I think is the intention).